src/vivarium/framework/values/post_processors.py

```diff
@@ -25,7 +25,9 @@ def __call__(self, value: Any, manager: ValuesManager) -> pd.Series[Any] | pd.Da
         ...
 
 
-def rescale_post_processor(value: NumberLike, manager: ValuesManager) -> NumberLike:
+def rescale_post_processor(
+    value: pd.Series[Any] | pd.DataFrame, manager: ValuesManager
+) -> pd.Series[Any] | pd.DataFrame:
```

Comment:
Are these types really Any? They're not `[int | float]`? In fact the return type looks like it will always be a float, since you're dividing by 365.0

Reply:
Oh, I guess for this particular post processor yeah probably just floats?